test/pagespeed_test.conf.template

```diff
@@ -1078,6 +1078,57 @@ http {
     pagespeed DisableFilters remove_comments,add_instrumentation;
   }
 
+  # Start remote config servers.
+  server {
+    listen @@SECONDARY_PORT@@;
+    listen [::]:@@SECONDARY_PORT@@;
+    server_name remote-config.example.com;
+    pagespeed FileCachePath "@@SECONDARY_CACHE@@";
+    pagespeed RemoteConfigurationUrl "http://127.0.0.1:9991/remote.cfg";
```

Comment:
In the apache test you made these ports be substituted variables; want to do that here too?